downstream/modules/hub/proc-downloading-signature-public-keys.adoc

```diff
@@ -19,7 +19,7 @@ The Signature Keys dashboard displays a list of multiple keys: collections and c
 
 . Choose one of the following methods to download your public key:
 
-* Select the menu icon and click btn:[Download Key] to download the public key.
+* Select the menu icon labeled btn:[Download Key] to download the public key.
```

Comment:
Suggest changing this to:

Select the btn:[Download Key] icon to download the public key. 

The label is the name of the icon.

Reply:
updated